webapp/src/components/github_issue_selector.jsx

```diff
@@ -41,7 +41,15 @@ export default class GithubIssueSelector extends PureComponent {
         const textEncoded = encodeURIComponent(text.trim().replace(/"/g, '\\"'));
 
         return Client.searchIssues(textEncoded).then((data) => {
-            return Array.isArray(data) ? data.map((item) => ({value: item, label: item.title})) : [];
+            return Array.isArray(data) ?
```

Comment:
```suggestion
           if (!Array.isArray(data)) {
               return [];
           }
           return data.map(...)
```

Reply:
I like this, it's more readable! thanks @mickmister